README.md

```diff
@@ -1,7 +1,7 @@
 ![Fireworq][logo]
 =================
 
-[![Build Status](https://travis-ci.org/fireworq/fireworq.svg?branch=master)](https://travis-ci.org/fireworq/fireworq) [![Coverage Status](https://coveralls.io/repos/github/fireworq/fireworq/badge.svg?branch=master)](https://coveralls.io/github/fireworq/fireworq?branch=master)
+[![Build Status](https://travis-ci.org/fireworq/fireworq.svg?branch=master)](https://travis-ci.org/fireworq/fireworq) [![Coverage Status](https://coveralls.io/repos/github/fireworq/fireworq/badge.svg?branch=master)](https://coveralls.io/github/fireworq/fireworq?branch=master) [![Docker Hub](https://img.shields.io/badge/docker%20build-ready-blue)](https://hub.docker.com/r/fireworq/fireworq) [![Latest version of on Docker Hub](https://images.microbadger.com/badges/version/fireworq/fireworq.svg)](https://microbadger.com/images/fireworq/fireworq)
```

Comment:
We could use ![](https://img.shields.io/docker/automated/fireworq/fireworq.svg) for the Docker Hub badge but it looks misleading that the image is actually built automatically by GitHub Actions.